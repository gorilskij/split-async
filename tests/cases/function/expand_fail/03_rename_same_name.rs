#![no_main]

use split_async::split;

#[split(a, a)]
async fn func() {}
