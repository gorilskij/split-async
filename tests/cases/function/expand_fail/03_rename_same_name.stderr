error: repeated identifier
 --> tests/cases/function/expand_fail/03_rename_same_name.rs:5:12
  |
5 | #[split(a, a)]
  |            ^
